index.html

```diff
@@ -23,7 +23,7 @@
           <li><a class="nav-link" href="#">Contact</a></li>
         </ul>
         <button class="hamburger">
-          <img src="./assets/icons/ic_menu.svg" alt="menu" />
+          <img src="./assets/icons/ic_menu.svg" alt="There was menu-icon" />
```

Comment:
Kindly note that the alt text shouldn't be too long. It should be brief as described [here](https://www.w3.org/WAI/test-evaluate/preliminary/#images)
![image](https://user-images.githubusercontent.com/73714615/188654813-8bbabc4c-b434-4c15-9be5-69111a9d6457.png)

A better alt text could be `menu-icon`